app/src/main/java/com/example/drawingactivity/ColorUtils.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.drawingactivity
+
+import java.lang.Integer.parseInt
+
+object ColorUtils {
+
+    /**
+     * Function that returns a boolean based on if the string matches a hexadecimal pattern.
+     */
+    fun isHexadecimal(input: String): Boolean {
+        return if (input.contains("#")) {
+            try {
+                parseInt(input, 16)
```

Comment:
Will this ever work? Will "#000000" parse with the # left on there?

Reply:
implemented a better if statement and substring to make sure this doesn't crash.